src/tests/EndToEndSpec.scala

```diff
@@ -109,12 +109,12 @@ case class EndToEndSpec()(implicit runner: Runner) {
       }
   }
 
-  // test("fetch entities - simple")(
-  //   Dao[TestSimpleEntity].all().toVector.sortBy(_.id)
-  // ).assert(_ == (simpleEntities ++ batchedSimpleEntities).sortBy(_.id))
-  // test("fetch entities - complex - long id")(
-  //   Dao[TestComplexLongId].all().toVector.sortBy(_.id)
-  // ).assert(_ == longIdComplexEntities.sortBy(_.id))
+  test("fetch entities - simple")(
```

Comment:
I would just make one tiny change to include `()` after every call to `run`, just to indicate to the reader that it's a side-effecting method.